hq/README.md

```diff
@@ -60,53 +64,47 @@ See `watched-account` template under `cloudformation` folder for the security po
 
 ### nginx setup
 
-At this step we configure nginx.
+1. Install dev-nginx:
+
+[dev-nginx](https://github.com/guardian/dev-nginx) contains a number of generic useful scripts to:
+- issue certs locally (and automatically trust them)
+- [generate an nginx config file](https://github.com/guardian/dev-nginx#setup-app) from a yaml definition
 
-1. Install nginx:
-	- Linux: `sudo apt-get install nginx`
-	- Mac OSX: `brew install nginx`
+MacOS users:
 
-2. Make sure you have a sites-enabled folder under your nginx home. This should be:
-	- Linux: `/etc/nginx/sites-enabled`
-	- Mac OSX: `/usr/local/etc/nginx/sites-enabled`
+```bash
+brew tap guardian/homebrew-devtools
+brew install guardian/devtools/dev-nginx
+```
+
+There are further instructions in the [dev-nginx](https://github.com/guardian/dev-nginx) repo.
 
-3. Ensure that your nginx.conf (found in the same location as the sites-enabled folder) contains the line `include sites-enabled/*` within the http block, for example:
+2. configure dev-nginx:
 
-  ```
-  http {
-      include sites-enabled/*;
-      ...
-  }
-  ```
+run each of these commands:
 
-4. Checkout the [dev-nginx](https://github.com/guardian/dev-nginx) repo onto your machine and follow the directions regarding installing and trusting the certificates (see [dev-nginx readme](https://github.com/guardian/dev-nginx)).
+`dev-nginx add-to-hosts-file`
 
-5. Install the nginx config for the security-hq application:
+`dev-nginx setup-cert security-hq.local.dev-gutools.co.uk`
 
-  ```
-    cd <path_of_dev-nginx>
-    sudo ./setup-app.rb <path_of_security-hq>/nginx/nginx-mapping.yml
-  ```
+`dev-nginx setup-app nginx/nginx-mapping.yml`
```

Comment:
[`setup-app` will issue the cert](https://github.com/guardian/dev-nginx/blob/master/script/setup-app#L109), it will also [restart nginx](https://github.com/guardian/dev-nginx/blob/master/script/setup-app#L115). That is, in this instance, it should be enough to only run `setup-app`. This is definitely not clear in the docs though and arguably is the wrong behaviour for `dev-nginx`?